src/epub/text/chapter-34.xhtml

```diff
@@ -52,10 +52,10 @@
 			<p>One morning at breakfast, Diana, after looking a little pensive for some minutes, asked him, “If his plans were yet unchanged.”</p>
 			<p>“Unchanged and unchangeable,” was the reply. And he proceeded to inform us that his departure from England was now definitively fixed for the ensuing year.</p>
 			<p>“And Rosamond Oliver?” suggested Mary, the words seeming to escape her lips involuntarily: for no sooner had she uttered them, than she made a gesture as if wishing to recall them. <abbr>St.</abbr> John had a book in his hand⁠—it was his unsocial custom to read at meals⁠—he closed it, and looked up.</p>
-			<p>“Rosamond Oliver,” said he, “is about to be married to <abbr>Mr.</abbr> Granby, one of the best connected and most estimable residents in S-, grandson and heir to Sir Frederic Granby: I had the intelligence from her father yesterday.”</p>
+			<p>“Rosamond Oliver,” said he, “is about to be married to <abbr>Mr.</abbr> Granby, one of the best connected and most estimable residents in S⸺, grandson and heir to Sir Frederic Granby: I had the intelligence from her father yesterday.”</p>
 			<p>His sisters looked at each other and at me; we all three looked at him: he was serene as glass.</p>
 			<p>“The match must have been got up hastily,” said Diana: “they cannot have known each other long.”</p>
-			<p>“But two months: they met in October at the county ball at S-. But where there are no obstacles to a union, as in the present case, where the connection is in every point desirable, delays are unnecessary: they will be married as soon as S⁠—Place, which Sir Frederic gives up to them, can he refitted for their reception.”</p>
+			<p>“But two months: they met in October at the county ball at S⸺. But where there are no obstacles to a union, as in the present case, where the connection is in every point desirable, delays are unnecessary: they will be married as soon as S⁠⸺Place, which Sir Frederic gives up to them, can he refitted for their reception.”</p>
```

Comment:
Good catch! Can you add a space after this one?

Reply:
The instance of `S--Place`

Reply:
Yep, will do

Reply:
Done